modules/decision/decision.py

```diff
@@ -0,0 +1,109 @@
+"""
+Creates decision for next action based on current state and detected pads
+"""
+
+from .. import decision_command
+from .. import object_in_world
+from .. import odometry_and_time
+
+
+class Decision:
+    """
+    Weighs distance to located pads and variance to choose the next action, either to land or move to the nearest pad
+    """
+
+    def __init__(self, tolerance: float):
+        self.__best_landing_pad = None
+        self.__weighted_pads = []
+        self.__distance_tolerance = tolerance
+
+    @staticmethod
+    def distance_to_pad(
+        pad: object_in_world.ObjectInWorld,
+        current_position: odometry_and_time.OdometryAndTime,
+    ):
+        """
+        Calculate Euclidean distance to landing pad based on current position.
+        """
+        dx = pad.position_x - current_position.odometry_data.position.north
+        dy = pad.position_y - current_position.odometry_data.position.east
+        return (dx**2 + dy**2) ** 0.5
```

Comment:
Square root unnecessary since it is a monotonic function (i.e. if sqrt(a) < sqrt(b) then a < b, for all non negative real a, b ).